validator/validator-main.protoascii

```diff
@@ -3376,7 +3359,6 @@ attr_lists: {
   attrs: { name: "[size]" }
   attrs: { name: "[spellcheck]" }
   attrs: { name: "[step]" }
-  attrs: { name: "[type]" }
```

Comment:
Did you intent to remove this? This will have an effect on non-email usages, potentially breaking pages.

Reply:
I did this intentionally Greg and what I did was add it back as an allowed attribute for AMP and AMP4ADS html formats, as seen in # 4.10.5, scroll further down and you will see the following change.

 # 4.10.5 The input element
 tags: {
   html_format: AMP
   html_format: AMP4ADS
   tag_name: "INPUT"
   ...
   attr_lists: "input-common-attr"
   attr_lists: "input-name-attr
   attrs: {name: "[type]"}  # Here I add it back as an allowed type

My reasoning behind this is because I wanted to reuse the attribute list "input-common-attr" for the tag type INPUT specific to AMP4Email, which would not allow for the "[type]" attribute.  Note also that I made the rule set for "INPUT" specific to AMP and AMP4ADS, as not doing so applied it to the AMP4Email html_format. I am wondering if there was a way to blacklists specific attributes but from what I've seen in the current rule set, we take the whitelist approach and only blacklist attribute values. Let me know if this is correct. Thanks for the review.